src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -106,6 +108,13 @@ class ChatSwitcherView extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        // Checks if the isChatSwitcherActive prop changed from false to true
+        // If the change happens, update the search value and focus on the input
+        if (!prevProps.isChatSwitcherActive && this.props.isChatSwitcherActive) {
+            this.updateSearch(this.state.search);
+            this.textInput.focus();
+        }
```

Comment:
I think  merging master will help with this since we are conditionally rendering the view now instead of toggling the active state of the "chat switcher".

Reply:
Noow I understand what you mean 🤦‍♂️

This is really not necessary anymore. Just calling ChatSwitcher.show() does the whole trick now.

Reply:
I believe there is still something to fix on this conditional render @marcaaron.
When the ChatSwitcher **is active**, and `ChatSwitcher.show()` is called, the text input does not focus again.

![bug](https://i.ibb.co/yNLZ0bV/croped-bug.gif)